### ISSUE TITLE
QA Ph3_Accessibility: Attending - Missing alternative text 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#638]
There are 3 cases of missing alternative text on the Attending pages when using the WAVE extension for Chrome, set to filter WCAG 2.0 AA. The same errors persist across the Event and Travel tabs in Attending. 

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Select SAttending
3. Enable the WAVE extension
4. View Style errors with the filter set to WCAG 2.0 AA 
5. Note missing alternative text errors 

Chrome 48 Mac OS X 10.11

![gio_638](https://cloud.githubusercontent.com/assets/15128124/14444857/62ff8466-fffd-11e5-8ccc-1a7300bfa3a6.png)
